isisd/isis_adjacency.c

```diff
@@ -207,6 +207,22 @@ static const char *adj_state2string(int state)
 	return NULL; /* not reached */
 }
 
+static const char *adj_level2string(int level)
+{
+	switch (level) {
+	case 1:
+		return "level-1";
+	case 2:
+		return "level-2";
+	case 3:
+		return "level-1-and-2";
```

Comment:
I see a bunch of cut-n-paste code that has the string as level-1-2.  I don't care what we use, but consistency would be prefered from my perspective

Reply:
and one L1L2.